stdlib/2and3/numbers.pyi

```diff
@@ -70,7 +70,11 @@ class Real(Complex, SupportsFloat):
         @abstractmethod
         def __ceil__(self) -> int: ...
         @abstractmethod
-        def __round__(self, ndigits: Optional[int] = ...): ...
+        @overload
+        def __round__(self, ndigits: None = ...): ...
```

Comment:
Not sure why this needs overloads.

Reply:
The idea is to have the same form as `float` above. Once this stub is filled the return types of the two overloads will be different, I think.